lms/routes.py

```diff
@@ -5,6 +5,8 @@ def includeme(config):
     config.add_route("assets", "/assets/*subpath")
     config.add_route("status", "/_status")
     config.add_route("favicon", "/favicon.ico")
+    config.add_route("ui-playground-root", "/ui-playground")
```

Comment:
I did not want a trailing slash to be a requirement so I added a separate route w/o the slash, I was not sure if there was a simpler way to do this. This also requires a second view :\

Reply:
Reading https://stackoverflow.com/a/15708788/434243 it looks to me like the idiomatic thing here is to declare only the route with a trailing slash (the one with the `/ui-playground/*subpath` pattern) and then handle requests for `/ui-playground` by making it redirect to the slash-appended route. The `/admin/` route already does this. You can access it at `/admin` and `/admin/`. Have a look in `admin.py` to see how this works.